src/plugins/publisher/CMakeLists.txt

```diff
@@ -4,6 +4,6 @@ ign_gui_add_plugin(Publisher
   QT_HEADERS
     Publisher.hh
   TEST_SOURCES
-    # Publisher_TEST.cc
+#    Publisher_TEST.cc
```

Comment:
Did you mean to edit the `Publisher` files?

Reply:
no no i was trying to run it to see something and i commented it again :D
i will modify that